packages/api/src/utils/db-client.js

```diff
@@ -10,6 +10,159 @@ export class DBClient {
         Accept: '*/*',
       },
     })
+    this._clientNftStorage = new PostgrestClient(endpoint, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+        Accept: '*/*',
+      },
+      schema: 'nftstorage',
```

Comment:
So the client needs the schema, I ended up needing to have two instances internally: auth + permaCache